sass/_ResponsiveFacets.scss

```diff
@@ -1,5 +1,12 @@
 @import 'bourbon/bourbon';
 @import 'Variables';
+
+body.coveo-block-scrolling {
```

Comment:
Why is the `body` portion of the selector needed?

Reply:
Good point, it's unnecessary. I'll remove it.